src/feed.njk

```diff
@@ -2,17 +2,17 @@
 pagination:
   data: config.languages
   size: 1
-  alias: language
-permalink: "{% if language !== config.defaultLanguage %}/{{ config.languages[language].slug }}{% endif %}/feed.xml"
+  alias: locale
+permalink: "{% if locale !== defaultLanguage %}/{{ config.languages[locale].slug }}{% endif %}/feed.xml"
```

Comment:
Can the `generatePermalink` function be used for this?

Reply:
I don't think so, actually — `generatePermalink` has to be called within an `11tydata.js` file's `eleventyComputed` block and I don't think adding a JavaScript data file just for the `feed.njk` file makes sense.